Apps/DBMaintainer/DrugMaintainerApp.cs

```diff
@@ -21,27 +21,26 @@ namespace HealthGateway.DrugMaintainer
     using HealthGateway.Common.Database.Models;
     using HealthGateway.Common.FileDownload;
     using HealthGateway.Common.Database;
-    using HealthGateway.DrugMaintainer.Database;
     using Microsoft.Extensions.Logging;
     using Microsoft.Extensions.Configuration;
     using System.IO;
     using System.IO.Compression;
 
     public class DrugMaintainerApp
     {
-        private ILogger logger;
+        private readonly ILogger logger;
         private IDrugProductParser parser;
         private IFileDownloadService downloadService;
-        private IDBContextFactory dbContextFactory;
-        private IConfiguration configuration;
+        private readonly IConfiguration configuration;
+        private readonly DrugDBContext drugDBContext;
```

Comment:
I thought we where going to be passing the IDbSet or the DbContext instead of the implementation.

Reply:
IDbSet would be within the context and was actually obsolete as it is part the the old EntityFramework.  I may be missing something on the DbContext, we could definitely pass it as that but then we would still need to reference the DrugDBContext to actually save the entities.